airflow/utils/python_virtualenv.py

```diff
@@ -88,12 +124,15 @@ def prepare_virtualenv(
     :type system_site_packages: bool
     :param requirements: List of additional python packages
     :type requirements: List[str]
-    :return: Path to a binary file with Python in a virtual environment.
+    :param connection_id: The private repository connection_id
+     case there are private packages to install.
```

Comment:
```suggestion
    :param connection_id: The connection ID of an http/https connection to use as the index url for installing
        pacakges from a private PyPi repository
```

Reply:
a private PyPI repository -> a private PEP 503 Simple Repository server, maybe? The term PyPI is confusing (officially it only refers to the “official” repository at PyPI.org)

Reply:
Oh yeah. I was also wondering and struggled with the naming. Thanks for explaining :)